pkg/manifestgen/install/install_test.go

```diff
@@ -41,5 +41,10 @@ func TestGenerate(t *testing.T) {
 		t.Errorf("toleration key '%s' not found", opts.TolerationKeys[0])
 	}
 
+	warning := GetGenWarning(opts)
+	if !strings.Contains(output.Content, GetGenWarning(opts)) {
```

Comment:
Given we know the position it should be in (at the start), I think it would be better to do:

```suggestion
	if !strings.HasPrefix(output.Content, GetGenWarning(opts)) {
```

Reply:
Ahh yep good call! Brain always immediately goes for `strings.Contains’ 😄